content/faq.md

```diff
@@ -8,6 +8,10 @@ Yes. We worked with a team of talented and specialized IP lawyers to create the
 ## Is the Hippocratic License open source?
 It is the intention and belief of the author that the Hippocratic License is an open source license, and that it meets the criteria of the [Open Source Definition](https://opensource.org/docs/osd) (OSD). However, it has not yet been reviewed by the OSI, so a determination on its OSI approval status is pending.
 
+## Is the Hippocratic License based on another open source license?
```

Comment:
Since the HL isn't (yet!) recognized as an open source license, I suggest "Is the Hippocratic License based on an existing open source license?"